documents/2-RTB/Documentazione_Esterna/Analisi_dei_Requisiti.typ

```diff
@@ -0,0 +1,45 @@
+#import "/template/document.typ": *
+
+#show: project.with(
+  title: "Analisi dei Requisiti",
+  managers: p.belenkov,
+  recipients: (
+    p.vardanega,
+    p.cardin,
+    p.azzurro,
+  ),
+  changelog: (
+    "0.0.1","2024-11-16", p.belenkov, p.santi,
+    [
+      Stesura iniziale del documento e breve introduzione.
+    ]
+  ),
+  show_images_list: false,
+  show_tables_list: false,
+  outline_depth: 3,
+)
+
+= *Introduzione*
+== Scopo del documento
+L'analisi dei requisiti è un documento fondamentale per tutti i progetti di sviluppo software che vogliono creare un prodotto a regola d'arte.\
+
+Lo scopo di questo documento è di definire le funzionalità che il sistema sarà in grado di offrire, ossia i requisiti obbligatori e opzionali che dovranno essere soddisfatti al fine di realizzare le richieste fatte dal proponente.\
+
+Il documento non si pone come una soluzione tecnica al problema, quanto più una definizione chiara e concisa di esso e di come possa essere risolto.\
+
+In particolare, le finalità di questo documento possono essere descritte nei seguenti punti:\
+
+-  *Definire le esigenze del proponente*:\
+Questo documento si basa principalmente sulle richieste del proponente, ossia le idee che quest'ultimo ha riguardo a come dovrebbe essere il software che verrà sviluppato da noi. Tali idee verranno raccolte tramite i vari documenti e incontri con #box(image(g.azzurro, width: 5em, height: auto)) che avverranno lungo il percorso dello svolgimento del progetto.\
```

Comment:
Perdonami, prima mi sono scordato di scrivere che anche il "sviluppato da noi" secondo me andrebbe modificato. Potresti scrivere "sviluppato dal nostro team" o "dal team" in generale (o gruppo alternativamente).
Il resto è perfetto, dopo questa modifica chiuderò la PR

Reply:
fixato